Jenkinsfile

```diff
@@ -65,6 +65,9 @@ pipeline {
               withDockerContainer(image: 'couchdbdev/centos-6-erlang-18.3', args: '-e LD_LIBRARY_PATH=/usr/local/bin --user 0:0') {
                 sh '''
                   cwd=$(pwd)
+                  rm -rf /tmp/couchjslogs
+                  mkdir -p /tmp/couchjslogs
+                  export COUCHDB_IO_LOG_DIR=/tmp/couchjslogs
```

Comment:
I had it there at first but noticed the tests were run in a `withDockerContainer` clause and wasn't sure environment vars would propagate into that environment